include/eve/detail/function/simd/arm/sve/load.hpp

```diff
@@ -0,0 +1,34 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/as.hpp>
+#include <eve/concept/vectorizable.hpp>
+#include <eve/detail/category.hpp>
+#include <eve/detail/implementation.hpp>
+#include <eve/detail/spy.hpp>
+#include <eve/module/core/regular/unalign.hpp>
+
+namespace eve::detail
+{
+
+template<real_scalar_value T, typename N, simd_compatible_ptr<wide<T, N>> Ptr>
+EVE_FORCEINLINE wide<T, N>
+                load_(EVE_SUPPORTS(sve_),
+                      ignore_none_ const&,
+                      safe_type const&,
+                      eve::as<wide<T, N>> const&,
+                      Ptr p) requires(sve_abi<abi_t<T, N>>)
+{
+  auto ptr = unalign(p);
+
+  if constexpr( N() == expected_cardinal_v<T> ) return svld1(svptrue_b8(), ptr);
+  else return svldff1(svptrue_b8(), ptr);
```

Comment:
I am using a fault tolerant load. 
The correct case will work. But maybe we should ensure that the asan will trigger on accessing garbage memory? I mean - this is a safe load.

Well - maybe later

Reply:
Sounds OK for now